clientgui/sg_PanelBase.h

```diff
@@ -35,9 +35,6 @@
 #define LARGE_FONT 16
 #endif
 
-#define SIDEMARGINS ADJUSTFORXDPI(30)
```

Comment:
I encourage you to restore this #define (eliminating the ADJUSTFORXDPI()) of course, thus:
`#define SIDEMARGINS 30
`
and please restore the use of `SIDEMARGINS ` in all the places you have changed  it to a literal `30`. It is important that this value be the same in all those places. It also makes it easier to change the margins if it is ever desired to do so in the future.
This value is uses in sg_BoincSimpleFrame.cpp, sg_PanelBase.cpp, sg_ProjectPanel.cpp and sg_TaskPanel.cpp, and must be the same in all those files.

Reply:
@CharlieFenton, I'll restore this and all others but change them to `const` instead of `#define` if you don't mind and have no strong arguments to use exactly macros instead of constants.